### PR TITLE
Iterate on "Scratch" puzzle type

### PR DESCRIPTION
**Pull request by joshlory**
Continue to iterate on https://github.com/code-dot-org/code-dot-org/pull/16703.

### CODE REVIEW

File: apps/src/scratch/scratch.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { singleton as studioApp } from '@cdo/apps/StudioApp';
 import ScratchView from './ScratchView';
 import { scratchDefaultProject } from './scratchDefaultProject';
 
-const Scratch = window.Scratch = window.Scratch || {};
+const Scratch = window.__TestInterface__ = {};
```
Comment (Bjvanminnen) [inline apps/src/scratch/scratch.js]: __TestInterface__ is something we use for a variety of things. I think what you actually want is
```
const Scratch = {};
window.__TestInterface__ = window.__TestInterface__ || {};
window.__TestInterface__.Scratch = Scratch;
```
or if you want to be ES6y
```
window.__TestInterface__ = {
  ...window.__TestInterface__,
  Scratch
};

Comment (joshlory) [reply apps/src/scratch/scratch.js]: Hmm, I was only able to find one match and it seems like it shouldn't conflict: https://github.com/code-dot-org/code-dot-org/search?q=__TestInterface__

Am I missing some other usages?

Comment (Bjvanminnen) [reply apps/src/scratch/scratch.js]: ugh. Looks like sometimes we use __TestInterface__ and sometimes __TestInterface. The latter seems like the more common one.

Comment (joshlory) [reply apps/src/scratch/scratch.js]: Hah, I just realized my link above cut off the trailing "__".  There's also: https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=__TestInterface__&type=

**Comment by joshlory** [comment]
PTAL.

**Pull request merged**